src/BenchmarkDotNet/Engines/EngineResolver.cs

```diff
@@ -24,16 +25,16 @@ private EngineResolver()
             Register(AccuracyMode.MinIterationTimeCharacteristic, () => TimeInterval.Millisecond * 500);
             Register(AccuracyMode.MinInvokeCountCharacteristic, () => 4);
             Register(AccuracyMode.EvaluateOverheadCharacteristic, () => true);
-            Register(AccuracyMode.RemoveOutliersCharacteristic, job =>
+            Register(AccuracyMode.OutlierModeCharacteristic, job =>
             {
                 var strategy = job.ResolveValue(RunMode.RunStrategyCharacteristic, this);
                 switch (strategy)
                 {
                     case RunStrategy.Throughput:
-                        return true;
+                        return OutlierMode.OnlyUpper;
```

Comment:
@AndyAyersMS this change implements your request to remove the outliers that are above the upper fence (the ones below lower fence are now going to remain)

@AndreyAkinshin 👍